modules/test/playwright/tests/login-web/main/login.spec.ts

```diff
@@ -105,3 +106,30 @@ test('LPD-28406 Access to My Account is allowed after disabling access to it thr
 		);
 	}
 });
+
+test('LPD-55426 Test sign in button is disabled until page is fully loaded', async ({
+	context,
+	page,
+}) => {
+	await performLogout(page);
+
+	const cdpSession = await context.newCDPSession(page)
+    await cdpSession.send('Network.emulateNetworkConditions', 
+		{ 
+        connectionType: 'cellular3g',
+        downloadThroughput: (750 * 1024) / 8,
+		latency: 50,
+		offline: false,
+		uploadThroughput: (250 * 1024) / 8,
+	});
+
+	await page.goto(liferayConfig.environment.baseUrl+'/c/portal/login');
+	await page.waitForLoadState('domcontentloaded');
+	expect(page.getByRole('button', {name: 'Sign In'})).toBeEnabled();
+
+	await page.goto(liferayConfig.environment.baseUrl);
+	await page.waitForLoadState('domcontentloaded');
+
+	await page.getByRole('button', {name: 'Sign In'}).click();
+	expect(page.getByRole('button', {name: 'Sign In'})).toBeEnabled();
```

Comment:
I don't get the test: shouldn't we expect that before the domcontentloaded state the button should be disabled and after enabled?
I mean... something like this (not tested so I really don't know if it's possible to do this 😄 )

```
await page.goto(liferayConfig.environment.baseUrl+'/c/portal/login');
expect(page.getByRole('button', {name: 'Sign In'})).toBeDisabled();
await page.waitForLoadState('domcontentloaded');
expect(page.getByRole('button', {name: 'Sign In'})).toBeEnabled();

await page.goto(liferayConfig.environment.baseUrl);
expect(page.getByRole('button', {name: 'Sign In'})).toBeDisabled();
await page.waitForLoadState('domcontentloaded');
expect(page.getByRole('button', {name: 'Sign In'})).toBeEnabled();
```

Reply:
I tried checking if the button is disabled before the domcontenloaded but playwright failed at that point saying that the button was enabled

Reply:
I think I have the test doing what we need to. I'll run it on ci, just to be sure :slightly_smiling_face: